draft-ietf-dnsop-structured-dns-error.md

```diff
@@ -473,7 +473,9 @@ whitespace, no blank lines) with ```'\'``` line wrapping per {{?RFC8792}}.
 # Security Considerations {#security}
 
 Security considerations in {{Section 6 of !RFC8914}} apply to this
-document.
+document, except the guard against using EDE content to alter DNS protocol
+processing. That guard is relaxed because the current specification mandates
+encryption, while {{!RFC8914}} assumes that EDE information is unauthenticated information.
```

Comment:
The specification also allows opportunistic privacy profile. We can say: 

The guard is relaxed in the current specification as it mandates encryption and recommends the use of an authenticated connection to the DNS server. However, {{!RFC8914}} assumes that EDE information is unauthenticated and sent over clear text.

We will also have to fix the following text:
OLD:
If the DNS client has enabled opportunistic privacy profile for DoT and the identity of the DNS server
cannot be verified, the DNS client MUST ignore the "c", "j", and "o" fields
but MAY process the "s" field and other parts of the response.
NEW:
If the DNS client has enabled opportunistic privacy profile for DoT and the identity of the DNS server
cannot be verified but the connection is encrypted, the DNS client MUST ignore the "c", "j", and "o" fields
but MAY process the "s" field and other parts of the response.

Reply:
Good catch. Updated the text with very minor edits.